conda_smithy/templates/run_docker_build.tmpl

```diff
@@ -26,10 +26,12 @@ show_channel_urls: true
 CONDARC
 )
 
+HOST_USER_ID=$( hash docker-machine 2>/dev/null && docker-machine ssh $(docker-machine active) 'id -u $(whoami)' || id -u $(whoami) )
```

Comment:
We could write a comment explaining why is that. It may not be clear for an OSX-only os Linux-only user